docs/content/integrations/deltalake/reference.mdx

```diff
@@ -403,7 +403,7 @@ aws dynamodb create-table --table-name delta_rs_lock_table \
 
 <TabItem name="Cloudflare R2 storage">
 
-Couldflare R2 storage has build in support for atomic copy operations. This can be leveraged by sending additional headers with the copy requrests.
+Clouddflare R2 storage has built-in support for atomic copy operations. This can be leveraged by sending additional headers with the copy requests.
```

Comment:
```suggestion
Cloudflare R2 storage has built-in support for atomic copy operations. This can be leveraged by sending additional headers with the copy requests.
```